src/user/views/adsManager/views/advanced/components/ads/AdField.tsx

```diff
@@ -171,7 +170,7 @@ function Preview({ name }: PreviewProp) {
       >
         <Box display="flex" flexDirection="row" justifyContent="center">
           <img
-            src={BraveLogo}
+            src="/brave_logo_icon.png"
```

Comment:
Are you sure this change works?  I don't see that png being processed by vite when I run `npm run build`, which I'd expect it to be. Doing the import of png's is the usual way to to signal to your packager that it needs to be packaged.

Reply:
I see, it was throwing errors but that was before the addition of the `env.d.ts`. After some experimentation, looks like they are all there